### ISSUE TITLE
pcl: Make error on buld target pcl_io

### ISSUE THREAD

**Issue by herx** [Homebrew/homebrew-science#1624]
==> cmake -DCMAKE_INSTALL_PREFIX=/usr/local/Cellar/pcl/1.7.2 -DCMAKE_BUILD_TYPE=
==> make

[ 11%] Built target pcl_sample_consensus
cd /tmp/pcl-0N1AYE/pcl-pcl-1.7.2/macbuild/io && /usr/local/Cellar/cmake/3.1.0/bin/cmake -E cmake_symlink_library ../lib/libpcl_io.1.7.2.dylib ../lib/libpcl_io.1.7.dylib ../lib/libpcl_io.dylib
/usr/local/Cellar/cmake/3.1.0/bin/cmake -E cmake_progress_report /tmp/pcl-0N1AYE/pcl-pcl-1.7.2/macbuild/CMakeFiles  37 38 39 40
[ 11%] Built target pcl_io
make: *** [all] Error 2

READ THIS: http://git.io/brew-troubleshooting
If reporting this issue please do so at (not Homebrew/homebrew):
  https://github.com/homebrew/homebrew-science/issues

**Comment by dpo** [comment]
There are several issues. First, `cmake` tries to detect if OpenMP is available. With clang, it isn't. But still the build fails because it's not finding `omp.h`. That should probably be reported upstream. Their [tutorial](http://www.pointclouds.org/documentation/tutorials/installing_homebrew.php#installing-homebrew) doesn't mention OpenMP.

The second issue is that if you build with `--cc=gcc-4.9`, it's not finding symbols of the form `Q_WS_X11`, which at QT symbols (again, `cmake` reports them as not found, but goes ahead and tries to compile and breaks down---that should probably be reported upstream too).

In discussion #1209, @fran6co suggests that building against QT5 would resolve the problem. Unfortunately, `cmake` then says
````
-- Found unsuitable Qt version "5.4.0" from /usr/local/opt/qt5/bin/qmake
````
So at this point, I think this should be discussed with the PCL folks.

**Comment by fran6co** [comment]
The OpenMP thing is an old issue and it doesn't stop the build because of it.

I'm rebuilding it to check what is wrong, probably is related to the boost update to 1.57 + QT4. I'll see if I can make a patch for it for 1.7.2 and a PR for master.

**Comment by fran6co** [comment]
QT5 is not ready yet => https://github.com/PointCloudLibrary/pcl/issues/936. But I have fixed this issue in https://github.com/Homebrew/homebrew-science/pull/1629

**Comment by yfwz100** [comment]
It seems that I came across this error, too. 

```
[ 11%] Built target pcl_sample_consensus
make: *** [all] Error 2
```

And it was fixed by #1629 .

**Comment by fran6co** [comment]
PR was merged, this issue can be closed.
